go/host/enclave/guardian.go

```diff
@@ -439,6 +439,17 @@ func (g *Guardian) submitL1Block(block *common.L1Block, isLatest bool) (bool, er
 			}
 			return g.submitL1Block(nextCanonicalBlock, isLatest)
 		}
+		// todo - Matt - is this the right way to do this?
+		// the issue is that sometimes a block was skipped and never resent
+		if strings.Contains(err.Error(), errutil.ErrBlockAncestorNotFound.Error()) {
```

Comment:
This shouldn't be necessary, the L1 catch-up logic handles this when it calls `L1Repo().FetchNextBlock(enclaveHead)` the L1 repo finds the LCA of L1 canonical head and the enclave's reported head.

Reply:
removed.